go.mod

```diff
@@ -136,7 +137,7 @@ replace (
 	github.com/microcosm-cc/bluemonday v1.0.2 => github.com/microcosm-cc/bluemonday v1.0.16
 	github.com/miekg/dns v1.0.14 => github.com/miekg/dns v1.1.25
 	github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.2.0
-	github.com/tendermint/tendermint => github.com/terra-money/tendermint v0.34.14-terra.3
+	github.com/tendermint/tendermint => github.com/classic-terra/tendermint v0.34.24-terra.0
```

Comment:
This version of tendermint contains a CPU griefing bug.

Reply:
Is this the CPU griefing issue your refering too? https://github.com/informalsystems/tendermint/issues/4

Reply:
If so I think it would make more sense to stay with 0.34.24 since the CPU griefing issue seems to be fixed in https://github.com/cometbft/cometbft/pull/245 so we should get this automagically when we release 2.2 and migrate to CometBFT